src/data/reducers/stafferInfo.test.js

```diff
@@ -1,11 +1,11 @@
-import * as actions from '../actions/instructorInfo';
+import * as actions from '../actions/stafferInfo';
```

Comment:
would be nice to have tests around the new stuff too

Reply:
✅